org.eclipse.jdt.ls.tests/projects/maven/invalid2/module1/pom.xml

```diff
@@ -0,0 +1,30 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>foo.bar</groupId>
+		<artifactId>multimodule</artifactId>
```

Comment:
Could we replace the references to "multimodule" with "invalid2" so they match the parent project folder name ?

Reply:
Fixed.